salt/modules/win_pkg.py

```diff
@@ -1217,31 +1217,37 @@ def install(name=None, refresh=False, pkgs=None, **kwargs):
             ret[pkg_name] = 'Unable to locate package {0}'.format(pkg_name)
             continue
 
-        version_num = options.get('version', '')
+        version_num = options.get('version')
```

Comment:
Are you saying that version will always exist under options and hence there is no need for a default value to prevent an error.

Reply:
No, just using the default `None`